src/Staker.sol

```diff
@@ -579,9 +611,9 @@ abstract contract Staker is INotifiableRewardReceiver, Multicall {
       scaledUnclaimedRewardCheckpoint: 0
     });
     _stakeTokenSafeTransferFrom(_depositor, address(_surrogate), _amount);
-    emit StakeDeposited(_depositor, _depositId, _amount, _amount);
-    emit ClaimerAltered(_depositId, address(0), _claimer);
-    emit DelegateeAltered(_depositId, address(0), _delegatee);
+    emit StakeDeposited(_depositor, _depositId, _amount, _amount, _earningPower);
+    emit ClaimerAltered(_depositId, address(0), _claimer, _earningPower);
+    emit DelegateeAltered(_depositId, address(0), _delegatee, _earningPower);
```

Comment:
This makes me think a separate event would be cleaner, but I guess this is more gas efficient

Reply:
Yeah, I went back and forth on it. I think I basically agree that a separate event would be cleaner, but it adds gas overhead, and at the end of the day is only a concern for integrators that can be managed with offchain code. So I landed here.